content/docs/start/data-pipelines.md

```diff
@@ -38,6 +38,8 @@ Now let's install the requirements:
 > first.
 
 ```dvc
+$ python3 -m venv .env
```

Comment:
this is great, didn't know that it's part of the standard library ... have been using `virtualenv -p python3 .env` so far

Reply:
Haha yeah it's awesome! `venv` is recommended since Python 3.6: [28.3. venv — Creation of virtual environments](https://python.readthedocs.io/en/stable/library/venv.html)